### DISCUSSION TITLE
Just built a 595 version of the TommyProm32 with a cheap nano clone (LGT8F328P) and it works !

### DISCUSSION THREAD

**Discussion by ABelliqueux** [TomNisbet/TommyPROM#73] [Show and tell]
For those looking to minimize cost, I built a [TommyProm32-595](https://github.com/TomNisbet/TommyPROM/pull/71) with a 3$ nano clone - a LGT8F328P-32 - and it works as intended. 

All I had to do was [install the LGT hardware definitions in the Arduino IDE](https://github.com/RalphBacon/LGT8F328P-Arduino-Clone-Chip-ATMega328P?tab=readme-ov-file#6-simple-steps-to-get-a-lgt8f328p-based-board-running-at-32mhz)  then set the board type to "LGT8F328P-LQFP32 MiniEVB", making sure to select the **16Mhz** variant. I was then able to flash the firmware. 

![TP32-595](https://github.com/user-attachments/assets/46120acb-77d4-449d-a25e-88024330e5a3)

**Comment by TomNisbet** [comment]
Looks great!
